source/office365/index.rst

```diff
@@ -0,0 +1,40 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+.. _office365:
+
+Using Wazuh to monitor Office 365
+=================================
+
+.. meta::
+  :description: Discover how Wazuh can help you to monitor your Office 365 organizations.
+
+.. versionadded:: 4.4.0
+
+This section provides instructions for monitoring the **Office 365** audit log for your organization.
+
+The **audit** log allows Wazuh to monitor:
+
+- User activity in SharePoint Online and OneDrive for Business
```

Comment:
Use a `.` to finish each item. Also, is this list complete or we could say that these are the most common events?